src/modules/infobox/components/ImportContent.vue

```diff
@@ -0,0 +1,254 @@
+                            @click="() => onChooseProvider(provider.value)"
+                        ></span>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div
+            class="connect-btn-container"
+            :class="{
+                disabled: isConnectDisabled,
+            }"
+        >
+            <button
+                type="button"
+                class="btn btn-outline-secondary connect-btn"
+                :disabled="isConnectDisabled"
+                @click="onConnect"
+            >
+                {{ buttonText }}
+            </button>
+        </div>
+        <!-- todo just for test -->
+        <div class="d-flex flex-column h-50 overflow-y-auto">
+            <div
+                v-for="layer in importedLayers"
+                :key="layer.externalLayerId"
+                @click="() => addLayer(layer)"
+                class="cursor-pointer"
+            >
+                {{ layer.name }}
+            </div>
+        </div>
+    </div>
+</template>
+<script>
+import externalLayerProviders from '@/external-layer-providers.json'
+import { isValidUrl, transformUrl } from '@/utils/url'
+import { mapActions, mapState } from 'vuex'
+import {
+    getCapWMSLayers,
+    getCapWMTSLayers,
+    isGpx,
+    isKml,
+    isWmsGetCap,
+    isWmtsGetCap,
+} from '@/utils/file'
+import { WMSCapabilities } from 'ol/format'
+import WMTSCapabilities from 'ol/format/WMTSCapabilities'
+import { optionsFromCapabilities } from 'ol/source/WMTS'
+import KMLLayer from '@/api/layers/KMLLayer.class'
+
+const BTN_RESET_TIMEOUT = 3000
+
+export default {
+    data() {
+        return {
+            importValue: '',
+            listShown: false,
+            // 'default | 'loading | 'failed' | 'succeeded'
+            uploadBtnStatus: 'default',
+            importedLayers: [], // todo just for test
+        }
+    },
+    computed: {
+        ...mapState({
+            lang: (state) => state.i18n.lang,
+        }),
+        filteredList() {
+            let providers = externalLayerProviders
+            if (this.importValue?.length) {
+                providers = externalLayerProviders.filter((it) => it.includes(this.importValue))
+            }
+            return providers.map((it) => {
+                return {
+                    value: it,
+                    toShow: it.replace(this.importValue, `<strong>${this.importValue}</strong>`),
+                }
+            })
+        },
+        isConnectDisabled() {
+            return !isValidUrl(this.importValue)
+        },
+        buttonText() {
+            if (this.uploadBtnStatus === 'loading') {
+                return this.$i18n.t('uploading_file')
+            } else if (this.uploadBtnStatus === 'failed') {
+                return this.$i18n.t('upload_failed')
+            } else if (this.uploadBtnStatus === 'succeeded') {
+                return this.$i18n.t('upload_succeeded')
+            }
```

Comment:
The following traduction keys seems better to me:
- `parsing_file`
- `parse_failed`
- `parse_succeeded`

Reply:
I used existing ones

Reply:
The traduction keys above already exists and I still think they are more appropriate for this button. The one you use (same on the old viewer) are more appropriate when uploading a file on the server, here we are loading (downloading) a file from a server into the application and then parsing it, therefore my proposition would be an improvement compared to the old viewer.

Reply:
ah ok, thanks, updated